### ISSUE TITLE
New Optifine 1.9 + WynnPack = missing tall grass texture..

### ISSUE THREAD

**Issue by SteveNorris** [Wynncraft/Issues#4722]
I have been excited for Optifine 1.9 coming out for weeks, meanwhile using Optifine 1.9 pre-release, which has worked completely fine with WynnPack. Now today when Optifine 1.9 came out I was finally able to build my OptiFine Minimap Profile, which works great except for this:

WITH NEW 1.9 OPTIFINE

![2016-04-01_12 29 27](https://cloud.githubusercontent.com/assets/18131746/14213330/6f47745e-f805-11e5-8abb-e08da40922be.png)

WITH PRE RELEASE 1.9 OPTIFINE

![2016-04-01_12 27 59](https://cloud.githubusercontent.com/assets/18131746/14213349/8445f826-f805-11e5-9daf-77af7fb732da.png)

Im using Optifine 1.9 (with shaders but had same issue w/o them) and Xaero's minimap (dunno why minimap would mess with textures though) through Forge 1.9.

I've been loving Wynncraft lately, after coming back for a new character going into Gavel, and now am highly enjoying the new HD Optifine and minimap. I got everything working together but this little missing texture makes it extremely hard to play (might be missing more but the grass is apparent).

**TL;DR** Using new 1.9 Optifine instead of 1.9 pre-release Optifine makes tall grass (possibly among other things) missing textures and **I am sad, wondering if there's any fix.**

**Comment by Odnod** [comment]
Is it like this on every server? Or is it specifically a wynncraft issue.

**Comment by SteveNorris** [comment]
Just Wynncraft.

**Comment by SteveNorris** [comment]
**UPDATE: **Narrowed it down; It is simply 1.9 "Beta" Forge causing the issue, mods or no, but there has to be a way to fix? Possibly even just deleting the tall grass texture?

**Comment by Trixter313** [comment]
Because this isn't an issue with the game and Wynncraft doesn't officially support or require any mods, I'm going to close this issue.

You might want to try posting to the Forge forums or just wait until Wynncraft is 1.9 only, that has a good chance of fixing it.
